packages/crawler/reports/login.porsche.com-login/2022-12-19_data-raw.json

```diff
@@ -0,0 +1,1088 @@
+{
+    "url": "https://login.porsche.com/login/de/de_DE",
+    "consumedPdsVersionsWithPrefixes": {
+        "2.17.0": [
+            ""
+        ]
+    },
+    "consumedTagNamesForVersionsAndPrefixes": {
+        "2.17.0": {
+            "": [
+                {
+                    "p-content-wrapper": {
+                        "properties": {
+                            "width": "fluid",
+                            "backgroundColor": "transparent",
+                            "theme": "light"
+                        },
+                        "slot": "<myaccess-login-background _ngcontent-serverapp-c85=\"\" _nghost-serverapp-c84=\"\">...</myaccess-login-background><p-flex _ngcontent-serverapp-c85=\"\" direction=\"column\" class=\"site hydrated\">...</p-flex>"
```

Comment:
- add comment in the code (implementation), why we added this
- replace "..." with some special text, for example "(the content was stripped)"

Reply:
done